protoc-gen-swagger/genswagger/template.go

```diff
@@ -821,7 +821,7 @@ func renderServices(services []*descriptor.Service, paths swaggerPathsObject, re
 					responseSchema.Ref = strings.Replace(responseSchema.Ref, `#/definitions/`, `#/x-stream-definitions/`, 1)
 				}
 				operationObject := &swaggerOperationObject{
-					Tags:       []string{svc.GetName()},
+					Tags:       []string{fmt.Sprintf("%s.%s", svc.File.GetPackage(), svc.GetName())},
```

Comment:
What will this produce if there is no package name?

Reply:
if there's no package name, there will be only a dot prepended to the service name, like `.svcName`. It seem to me that's fine if we assume the tag always consists of two parts: the package name and the service name. We could use `strings.Split` to extract the two parts even if the optional `package` name is not defined in the proto file.

Reply:
I think we should only add the package name if it is set. Could we just add a:

```go
tag := svc.GetName()
if pkg := svc.File.GetPackage(); pkg != "" {
    tag = pkg + "." + tag
}
```
?

Reply:
Sure, that's a good idea. Will do.